network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/Edge.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.iidm.impl;
+
+import com.powsybl.network.store.model.NodeBreakerBiConnectable;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class Edge {
+
+    private final NodeBreakerBiConnectable biConnectable;
+
```

Comment:
When we build a graph in bus/breaker, the `biconnectable` is a switch who notdefine getNode1 and getNode2
I think that either `biconnectable` must be independent of the type of the toplogy and also implement getBus1 and getBus2 or we must also have a class BusBreakerBiConnectable